### PR TITLE
Use the \url command instead of \href to display urls in bibliography…

### PR DESCRIPTION
**Pull request by cgogolin**
… so that they are broken across multiple lines correctly

I like your trick to supress the https:// part of the url but I am actually not sure whether the old version would work correctly with http only urls and if a url is long then \href does not break it across multiple lines.

Any objections against this fix @dwierichs ?

### CODE REVIEW

**Comment by k4rtik** [comment]
Is there a plan to provide the BibLaTeX version of this bib style? I think an interesting functionality missing in biblatex is support for the `code` field.

**Comment by dwierichs** [comment]
There is no plan for this so far, I think. I would be super happy for any contributions in this direction! Would you mind opening an issue requesting this? Then it won't be hidden in this pull request. :)

**Comment by cgogolin** [comment]
The problem with BibLaTeX is that (at leat the last time I checked the arxiv still wasn't able to run BibTeX or Biber and thus a locally generated bbl must be uploaded. Now if that was generated with a BibLaTeX version that doesn't match that of the arXiv things will fail with hard to understand errors.

But coming back to this PR: @dwierichs any objections against the changes here?

File: quantum.bst [hunk 1]
```diff
@@ -1147,10 +1147,10 @@ FUNCTION {format.url}
   shorturl empty$
     { url empty$
         { "" }
-        { " url:~\href{" url * "}{" * "https://" #8 url chop.word * "}" * }
+	{ " url:~\url{" url * "}" * }
       if$
     }
-    { " url:~\href{" url * "}{" * "https://" #8 shorturl chop.word * "}" * }
+    { " url:~\url{" url * "}" * }
```
Comment (dwierichs) [inline quantum.bst]: This effectively deactivates the `shorturl` functionality, doesn't it? And `\url` does not allow us to display more human-readable versions of the content... Could `breakurl` be an option here? I'll look into this.

**Review by dwierichs — COMMENTED** [review]
Let me take another look at our options, ok?

**Comment by dwierichs** [comment]
Okay, I thought a bit more about this. What about the following reasoning?
1. Breaking long URLs is a headache with `\href` and there does not seem to be a proper option to fix this.
2. We therefore use `\url` for this rather than `\href`
3. If users provide a `shorturl`, we assume that they take care of proper breaking/sufficiently small length of the link themselves and continue using `\href`. (The concept of a `shorturl` field is kind of incompatible with `\url` not providing an own print-out string but just the URL itself...)

**Comment by cgogolin** [comment]
> Okay, I thought a bit more about this. What about the following reasoning?
> 
> 1. Breaking long URLs is a headache with `\href` and there does not seem to be a proper option to fix this.
> 2. We therefore use `\url` for this rather than `\href`
> 3. If users provide a `shorturl`, we assume that they take care of proper breaking/sufficiently small length of the link themselves and continue using `\href`. (The concept of a `shorturl` field is kind of incompatible with `\url` not providing an own print-out string but just the URL itself...)

I like that approach.

File: quantum.bst (hunk 1 above)
Comment (cgogolin) [reply quantum.bst]: Yes it does.

Comment (dwierichs) [inline quantum.bst]:
```suggestion
    { " url:~\href{" url * "}{" * "https://" #8 shorturl chop.word * "}" * }
```

Comment (dwierichs) [reply quantum.bst]: As suggested in my other comment, reverting this part of the change that affects the `shorturl` field.

**Comment by cgogolin** [comment]
Thanks @dwierichs!

**Pull request merged**